cpp/src/aztec/honk/composer/standard_honk_composer.hpp

```diff
@@ -20,10 +20,11 @@ class StandardHonkComposer {
     // 1) Proving and verification keys
     // 2) CRS
     // 3) Converting variables to witness vectors/polynomials
-    ComposerHelper composer_helper;
+    ComposerHelper<StandardCircuitConstructor> composer_helper;
 
     // Leaving it in for now just in case
     bool contains_recursive_proof = false;
+    static constexpr size_t program_width = 3;
```

Comment:
Maybe this also belongs in flavor.hpp? That's a proto version of the fututure settings/arithmetization abstraction that I started in the sumcheck work because I needed to put some constants someplace.

Reply:
I moved the constant there, but the member is needed to make it easier to copy the tests (for example, permutation tests). This way the test can simply use the value defined in the class.